lesson_8/homework_8.py

```diff
@@ -0,0 +1,20 @@
+
+def sum_string(string):
+    try:
+        numbers = (int(num) for num in string.split(','))
+        return sum(numbers)
+    except ValueError:
+
+        return "Не можу це зробити!"
+
+
+strings = ["1,2,3,4", "1,2,3,4,50", "qwerty1,2,3"]
+
+
+for string in strings:
+    result = sum_string(string)
+    print(result)
```

Comment:
Better to collect results with list comprehension here and only after to do print